pkg/configuration/http_test.go

```diff
@@ -0,0 +1,50 @@
+package configuration
+
+import (
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/cloudtrust/common-service/v2/log"
+	"github.com/cloudtrust/keycloak-bridge/internal/keycloakb"
+	"github.com/cloudtrust/keycloak-bridge/pkg/configuration/mock"
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/mock/gomock"
+)
+
+func TestHTTPConfigurationHandler(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+	mockConfigurationComponent := mock.NewComponent(mockCtrl)
+
+	const (
+		realm             = "trustid"
+		contextKey        = "19251660-f869-11ec-b939-0242ac120002"
+		identificationURI = "http://identification-uri"
+	)
+
+	r := mux.NewRouter()
+	r.Handle("/configuration/realms/{realm}/identification", MakeConfigurationHandler(keycloakb.ToGoKitEndpoint(MakeGetIdentificationURIEndpoint(mockConfigurationComponent)), log.NewNopLogger()))
```

Comment:
You don't want to test all this package code at once. In unit test, you can write a simple handler that concatenate the realm and the context key or write them as a JSON map. This way, you can check that your HTTPConfigurationHandler correctly identify the realm and the context-key and you can check its behavior when the provided context-key does not match the UUID regex